doc/deploy_delta.md

```diff
@@ -0,0 +1,58 @@
+# Livepeer Delta upgrade deployment steps
+
+## Step 1: Deploy contracts
+
+Run the deployment script to deploy the new contracts:
+
+```
+npx hardhat deploy --tags DELTA_UPGRADE --network arbitrumMainnet
+```
+
+You must configure the private key for an account that has enough ETH to run the transactions.
+
+## Step 2: Prepare governance update
+
+1.  Grab the full git hash from the current repository where you ran the deployment script, or get it from the output of
+    the pending governance actions.
+2.  Also obtain the addresses of the contracts deployed above. You can find them in the corresponding `deployments` directory or
+    in the output of the deployment script.
+3.  Now on the `governance-scripts` repository, update:
+    a) the `0xPENDING_ADDRESS` address entries on the `updates/addresses.js` file with the deployed contracts.
+    b) the `0xPENDING_GIT_HASH` references in `updates/l2-lip-delta-91-92.js` with the git hash from the protocol
+    repository.
+
+## Step 3: Run governance update
+
+In the `governance-scripts` repository, run the governance update script:
+
+```
+node index.js create ./updates/l2-lip-delta-91-92.js 0
+```
+
+This will print out the transaction that should be run by the governor owner to stage the update
```

Comment:
Let's make it clear that the "protocol governor" is being referenced here and below to avoid confusion with the LivepeerGovernor being deployed for LIP-91/92.